scripts/oc-cluster.sh

```diff
@@ -12,15 +12,15 @@ sudo service docker start
 sudo service docker status
 
 ## download oc binaries
-sudo wget https://github.com/openshift/origin/releases/download/v3.10.0/openshift-origin-client-tools-v3.10.0-dd10d17-linux-64bit.tar.gz -O /tmp/openshift-origin-client-tools.tar.gz 2> /dev/null > /dev/null
+sudo wget https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz -O /tmp/openshift-origin-client-tools.tar.gz 2> /dev/null > /dev/null
```

Comment:
Like what @anmolbabu said, perhaps we can make this link into a constant / variable? Take the link and put it in a $OPENSHIFT_URL_DOWNLOAD or something.

Same with the bellow code too. If we do that, we can possibly change the version we want to test against in the future..

Reply:
+1 :+1:

Reply:
:+1: